src/Assets/scss/main.scss

```diff
@@ -911,9 +911,9 @@ body {
     color: #ff0000;
 }
 
-.loading-spinner-height {
-    height: 10em;
-    width: 10em;
+.loading-spinner-small {
```

Comment:
In Main Css we have two css for loading spinner height so i am using second for small spinner.